src/components/Observer/Observer.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef, useState } from 'react';
+import { StyledDiv } from './styles';
+
+export default function Observer({callback}: { callback: Function}) {
```

Comment:
React.FC (FunctionalComponent). The same for all other components of this project.